cvmfs/sync_mediator.cc

```diff
@@ -984,13 +984,6 @@ void SyncMediator::AddLocalHardlinkGroups(const HardlinkGroupMap &hardlinks) {
   for (HardlinkGroupMap::const_iterator i = hardlinks.begin(),
        iEnd = hardlinks.end(); i != iEnd; ++i)
   {
-    if (i->second.hardlinks.size() != i->second.master->GetUnionLinkcount()) {
-      LogCvmfs(kLogPublish, kLogStdout, "Hardlinks across directories (%s)",
```

Comment:
Here we would for the time being keep the possibility to be strict about hardlink treatment.  That means the `if` block should stay but the logging statement would log to `kLogDebug` instead of `kLogStdout`